docs/man/Dockerfile.5.md

```diff
@@ -131,7 +131,11 @@ or
  interactively, as with the following command: **docker run -t -i image bash**
 
 **ADD**
- --**ADD <src>... <dest>** The ADD instruction copies new files, directories
+ --ADD has two forms:
+ **ADD <src>... <dest>**
+ **ADD ["<src>"... "<dest>"]** This is the required form when paths contain
+ whitespaces.
```

Comment:
Suggest we simplify/clarify this to: "This form is required for paths containing whitespaces."